siliconcompiler/core.py

```diff
@@ -513,40 +527,42 @@ def find_function(self, modulename, funcname, moduletype=None):
 
         '''
 
+        # Ensure that SC built-ins are on the $PYTHONPATH (only necessary for local installs)
+        if not self.scroot in sys.path:
+            sys.path.append(self.scroot)
```

Comment:
Should we just do this in `__init__`?

Reply:
Probably - I also think that we may be able to remove it and replace all of the `tools.[...]` imports with `siliconcompiler.tools.[...]`